website/static/js/project.js

```diff
@@ -61,38 +60,8 @@ NodeActions.forkNode = function() {
                 isCors: true,
                 data: payload
             }
-        ).done(function(response) {
-            $osf.unblock();
-            bootbox.confirm({
-                message: '<h4 class="add-project-success text-success">Fork created successfully!</h4>',
-                callback: function(result) {
-                    if(result) {
-                        window.location = response.data.links.html;
-                    }
-                },
-                buttons: {
-                    confirm: {
-                        label: 'Go to new fork',
-                        className: 'btn-success'
-                    },
-                    cancel: {
-                        label: 'Keep working here'
-                    }
-                },
-                closeButton: false
-            });
-        }).fail(function(response) {
-            $osf.unblock();
-            if (response.status === 403) {
```

Comment:
It's assumed the user isn't going to be hanging around for the response, if we make AJAX wait on a response it won't complete the action. Also it's redundant client-side stuff because someone with out permission shouldn't even be able to click that button.

It's consistent if you imagine the forks taking a relatively long time to complete (like 5 minutes) nobody's going to want to stick around for 5 minutes to discover they got an error. But the failure email should have that info.